### PR TITLE
Add weos4 network platform to documentation

### PR DESCRIPTION
**Pull request by ernst-s**
##### SUMMARY

Add Westermo WeOS 4 network platform documentation

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

weos4

##### ADDITIONAL INFORMATION

The weos4 cliconf and terminal plugins were accepted today in community.network. This pull request updates the documentation for this new platform.

```paste below

```

### CODE REVIEW

File: docs/docsite/rst/network/user_guide/platform_weos4.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     when: ansible_network_os == 'community.network.weos4'
+
+Example Configuration task
+--------------------------
+
+.. code-block:: yaml
+
+   - name: Replace configuration with file on ansible host (WeOS 4)
+     ansible.netcommon.cli_config:
+       config: "{{lookup('file', 'westermo.conf')}}
```
Comment (kindlehl) [inline docs/docsite/rst/network/user_guide/platform_weos4.rst]: Missing end quote.

I think spaces make this more readable, so I'd do `"{{ lookup('file', 'westermo.conf') }}"`

File: docs/docsite/rst/network/user_guide/platform_weos4.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- If you are accessing your host directly (not through a bastion/jump host) you can remove the ``ansible_ssh_common_args`` configuration.
+- If you are accessing your host through a bastion/jump host, you cannot include your SSH password in the ``ProxyCommand`` directive. To prevent secrets from leaking out (for example in ``ps`` output), SSH does not support providing passwords via environment variables.
+
+Example CLI task
+----------------
+
+.. code-block:: yaml
+
+   - name: Get version information (WeOS 4)
+     ansible.netcommon.cli_command::
```
Comment (kindlehl) [inline docs/docsite/rst/network/user_guide/platform_weos4.rst]: Extra colon at the end

**Review by kindlehl — CHANGES_REQUESTED** [review]
Overall, this looks great and the layout is consistent with other networking guides. Some small  format issues should be addressed.

File: docs/docsite/rst/network/user_guide/platform_weos4.rst (hunk 2 above)
Comment (ernst-s) [reply docs/docsite/rst/network/user_guide/platform_weos4.rst]: Thanks for the review, I've applied the suggestions.

**Review by kindlehl — APPROVED** [review]
Looks good to me

**Review by samccann — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Looks good. We need to wait for the 2.2.0 release of community.network before we merge - 
https://galaxy.ansible.com/community/network

**Comment by Andersson007** [comment]
cc @dericcrago ^

**Comment by dericcrago** [comment]
hi all, @samccann, `community.network` `2.2.0` has been released (https://github.com/ansible-collections/community.network/releases/tag/2.2.0)

**Comment by acozine** [comment]
rebuild_merge

**Comment by Andersson007** [comment]
Thanks everyone!

**Pull request merged**